### PR TITLE
Service catalog rebase v0.1.0

### PR DESCRIPTION
**Pull request by jpeeler**
This is the beta service catalog release. Some of the building and image usage has slightly changed due to the use of a single binary, rather than two from before.

### CODE REVIEW

**Comment by pmorie** [comment]
@jpeeler I think you need to look at hack/test-cmd.sh locally - it's been flaking reliably and isn't something that has typically flaked out on us during these rebase PRs.

**Comment by jpeeler** [comment]
still debugging locally, but just in case it is a flake...
/test cmd

File: cmd/service-catalog/go/src/github.com/kubernetes-incubator/service-catalog/Makefile [hunk 1]
```diff
@@ -73,15 +74,13 @@ $(error Unsupported platform to compile for)
 endif
 
 GO_BUILD       = env GOOS=$(PLATFORM) GOARCH=$(ARCH) go build -i $(GOFLAGS) \
-                   -ldflags "-X $(SC_PKG)/pkg.VERSION=$(VERSION)"
+                   -ldflags "-X $(SC_PKG)/pkg.VERSION=$(VERSION) $(BUILD_LDFLAGS)"
```
Comment (stevekuznetsov) [inline cmd/service-catalog/go/src/github.com/kubernetes-incubator/service-catalog/Makefile]: Do you need to support Go 1.4? That version requires `-X A B` not `-X A=B`

**Comment by stevekuznetsov** [comment]
/hold

my LGTM should not be the final call here

**Comment by stevekuznetsov** [comment]
Wait no didn't see Paul's LGTM

/hold cancel

**Comment by stevekuznetsov** [comment]
@jpeeler congrats you get not one but TWO merges!!@!@!@

**Pull request merged**